docs/basics/userguide/seeding.md

```diff
@@ -2,9 +2,40 @@
 
 # How to seed your experiment
 
-Rlberry has a class [Seeder](rlberry.seeding.seeder.Seeder) that conveniently wraps a [NumPy SeedSequence](https://numpy.org/doc/stable/reference/random/parallel.html), and allows you to create independent random number generators for different objects and threads, using a single [Seeder](rlberry.seeding.seeder.Seeder) instance. It works as follows:
+
+## Why use seeding?
+First, maybe you don't know what seeding are, and why you should use it.
+
+The seeding is the process of initializing a random number generator with a specific value (the seed) to ensure that the sequence of random numbers it produces is repeatable and consistent.
+
+
+Here is 3 reason why seeding is important
+1. Reproducibility
+
+Seeding ensures that the sequence of random numbers (or any other stochastic process) generated by your algorithm is repeatable. This is essential for debugging, testing, and validating your algorithm.
+When testing an algorithm that uses random sampling, if you encounter an issue, you need to reproduce the exact sequence of operations that led to the issue. By using a fixed seed, you can guarantee that the same random numbers are generated each time, allowing you to trace and fix the problem consistently.
```

Comment:
This sentence is redundant with what is before no ?

Reply:
And also in addition to test and debug it is useful to have a third party (e.g. reviewer at a conference) independently reproduce your work to assert that it indeed works as advertised.

Reply:
done.